### PR TITLE
Feat/additional-configurability

### PR DESCRIPTION
**Pull request by kvnloughead**
Add configurability to button labels and form component.

### CODE REVIEW

File: src/config/buttons.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const labelsById = {
+  'watch-video': 'Watch Video',
```
Comment (hotcheetoqueen) [inline src/config/buttons.js]: can we make `watch-video` & `learn-more` id's generic like the others? if we're saying the client can add whatever text they want we should make these buttons flexible. when two buttons are next to each other, the common term is `primary-button` or `primary-cta` and `secondary-button`/`cta` or `ghost-button` (when it has that outline). there could be a future world where the client wants to show case something else in the header area.. eg partner site, podcast, blog, pr article, etc

**Review by hotcheetoqueen — CHANGES_REQUESTED** [review]

File: src/config/buttons.js (hunk 1 above)
Comment (kvnloughead) [reply src/config/buttons.js]: Absolutely, good suggestion.

**Comment by kvnloughead** [comment]
This PR will improve configurability for the project by

- making the button labels configurable 
- making the inputs of the form configurable
- moving configurable section-titles to the corresponding config/section.js files

File: src/config/buttons.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export const labelsById = {
+  'header-primary-button': 'Watch Video',
+  'header-secondary-button': 'Learn more',
+  carousel: 'Get more info',
```
Comment (hotcheetoqueen) [inline src/config/buttons.js:4]: out of curiosity, why are we making some button id's configurable but not others?

Comment (kvnloughead) [reply src/config/buttons.js:4]: Actually, none of them are intended to be configurable, but you can't have dashes in the keys without quotation marks.  Maybe I should change to camel case

Comment (hotcheetoqueen) [reply src/config/buttons.js:4]: oo, per our goal to tell the client they can feel safe changing all things in quotes in the config file, i think camel case so we can avoid quotes is a great idea!

**Review by hotcheetoqueen — APPROVED** [review]
i think we should keep all things in quotes in the config file editable by the client and use camel case or another solution for multi-word non-editable keys. once you sort how you want to update that (so long as that works for you as well!) then this is great to merge :)!

**Pull request merged**